typescript/src/lib/util/decorators.ts

```diff
@@ -85,20 +86,27 @@ export const requiresLevel = (level: PermissionLevels, fallback?: Fallback): Met
 	}, fallback);
 };
 
+const ServerOnlyPermissions = [Permissions.FLAGS.MANAGE_MESSAGES, Permissions.FLAGS.ADD_REACTIONS];
```

Comment:
Use `new Permissions`, so the array is resolved to the most efficient data structure.